packages/contracts-bedrock/src/L1/OptimismPortalInterop.sol

```diff
@@ -20,10 +20,9 @@ import { IL1BlockInterop, ConfigType } from "interfaces/L2/IL1BlockInterop.sol";
 contract OptimismPortalInterop is OptimismPortal2 {
     constructor(
         uint256 _proofMaturityDelaySeconds,
-        uint256 _disputeGameFinalityDelaySeconds,
-        address _sharedLockbox
+        uint256 _disputeGameFinalityDelaySeconds
```

Comment:
Is the contract version well after this change?

Reply:
Yeah cause it hasn't been merged to `develop` yet

Reply:
yes but I mean, we undo a change, shouldn't we undo the version?
Or is still has some change different to this?